### DISCUSSION TITLE
IOS release?

### DISCUSSION THREAD

**Discussion by GhostInTheKayak** [organicmaps/organicmaps#2535] [Q&A] [answered]
The last iOS release seems to be 2021.10.08 and since then, there have been 13 Android releases -- [see](https://github.com/organicmaps/organicmaps/tags)

Are there plans for another IOS release?

**Comment by biodranik** [comment, marked as answer]
The latest iOS release at the moment is [2022.04.26](https://apps.apple.com/app/organic-maps/id1567437057), we use the same android tag also for the iOS.
